Fix SafeConfigParser deprecation

ndokos opened the pull request:
Recent python3 has deprecated SafeConfigParser which now emits warnings. We import ConfigParser which is now what SafeConfigParser used to be.

portante commented on configtools.spec:
```diff
@@ -4,11 +4,16 @@ Release:     1
 Buildarch:   noarch
 
+%if 0%{?rhel} == 7
 BuildRequires:	python2-devel
```
Why does this work if we appear to require python3 now?

ndokos replied:
This is dealt with in PR#15 I believe. There were commits from that PR in the tree for this PR, which I have tried to separate out, so now the changes in PR#15 relative to master are separate from the changes in this PR relative to master (except for  some changes in .travis.yml: for that, the changes in this PR should win). I hope that makes it clearer.

portante commented on configtools/__init__.py:
```diff
@@ -5,10 +5,10 @@
-    from configparser import SafeConfigParser
+    from configparser import ConfigParser
 except:
     from ConfigParser import SafeConfigParser
```
Why even keep this option if we no longer support python2?

ndokos replied:
Got rid of the python2 tests in .travis.yml and simplified this as you suggest.

ndokos committed:
SafeConfigParser deprecation

SafeConfigParser has been deprecated and is now an alias for
ConfiParser, but will be removed in the future, so change all
instances of it to ConfigParser in the code.

Get rid of python2 and obsolete python3 versions in Travis.

portante reviewed (approved):
Looks good!  Ship it.
